lib/storage.js

```diff
@@ -39,17 +40,23 @@ function defaultStorage (storage, opts) {
   try {
     // Store in .dat with secret in ~/.dat
     if (fs.statSync(storage).isDirectory()) {
-      return datStore(storage)
+      if (typeof opts.secretDir !== 'undefined') {
+        fs.statSync(opts.secretDir).isDirectory()
+      }
+      return datStore(storage, opts)
```

Comment:
Do we need to return this here or can just return it after try/catch? (having trouble reading with the diff..